frontend/package.json

```diff
@@ -64,13 +64,12 @@
     "webpack-merge": "^4.1.4"
   },
   "scripts": {
-    "prebuild": "yarn clean",
-    "build": "webpack --config webpack.prod.js",
+    "build": "webpack --config webpack.dev.js",
```

Comment:
do we need to change this?

Reply:
Good question. It works with dev for sure. Need to check if it works with prod